### ISSUE TITLE
CollectionView cannot scroll when data overflows

### ISSUE THREAD

**Issue by shushu789** [dotnet-presentations/dotnet-maui-workshop#123]
Hello，I'm new to .NET MAUI, I watched your video on YouTube and I saw you said that you don't need to put the CollectionView inside the scroll control to scroll, but when I try to do this on Android emulator and Windows, it Doesn't work unless the HeightRequest attribute is added, and then I found this related question through the search engine, saying that this is a normal design?（[maui-issues-8888](https://github.com/dotnet/maui/issues/8888)） So I'm wondering why your demo in the video doesn't have this problem. Thank you.
-  Here is the source of the video and the location of the point in time relevant to the issue:
[CollectionView data can be scrolled when it overflows the visible range](https://youtu.be/DuNLR_NJv8U?t=3191)

-  This is my development environment:
.NET 8
visual studio 2022 17.9
Windows 10

**Comment by jamesmontemagno** [comment]
If you clone this repo, open the Finished folder project and run it does that work for you? It does for me and is scrolling fine. 

I tested it here on my windows machine and android phone.

I also opened up the Part 2 folder, which is the where the video you linked to is. I added more monkeys into the hardcoded list and launched it and is scrolling fine for me here...

**Comment by shushu789** [comment]
Thank you very much for getting back to me!
I apologize to you for not describing it clearly at first, what I meant was that scrolling doesn't work when the content is out of the visible range when the CollectionView is inside a VerticalStackLayout, like this:
```
 <VerticalStackLayout>
     <CollectionView>
         <! --... -->
     </CollectionView>
 </VerticalStackLayout>
```

This is also basically the same as described at the beginning of your YouTube video, but what you demonstrated in the video works, and what I tested doesn't.
I tested the Finished folder project, which uses a Grid instead of a VerticalStackLayout, and this won't be a problem because I also replaced it with a Grid the first time I encountered the problem above. like this:
```
 <Grid>
     <CollectionView>
         <! --... -->
     </CollectionView>
 </Grid>

```

Translated with DeepL.com (free version)
